site/src/pages/DeploymentSettingsPage/IdpOrgSyncPage/IdpOrgSyncPageView.tsx

```diff
@@ -118,6 +132,19 @@ export const IdpOrgSyncPageView: FC<IdpSyncPageViewProps> = ({
 		form.handleSubmit();
 	};
 
+	const handleKeyDown: KeyboardEventHandler<HTMLInputElement> = (e) => {
```

Comment:
idk when we started using `e` like this but I really don't like it

```suggestion
	const handleKeyDown: KeyboardEventHandler<HTMLInputElement> = (event) => {
```